### ISSUE TITLE
Adguard add log

### ISSUE THREAD

**Issue by Kugane** [mimugmail/opn-repo#228]
Hi, i had a few Problems with my adguard and searched for the log.
But there isnt one configured out of the box.

Maybe add some?
In adguard.yaml there is a logfile Option.

I'm actually on my phone 🙈 If needed iccan provide more info later.

Ty for your effort 👍🏼

**Comment by 8bitDesigner** [comment]
Yep, I'd 👍 this. For future reference, I was able to get logging by shelling into my OPNSense instance and editing `/usr/local/AdGuardHome/AdGuardHome.yaml` and setting the `file:` option under logging to `"/var/log/adguard.log"`.
